templates/login.tpl

```diff
@@ -14,7 +14,8 @@
 	</tr>
 	<tr>
 		<td class="label"><label>{$PALANG.password}:</label></td>
-		<td><input class="flat" type="password" name="fPassword" /></td>
+		<td><input class="flat" type="password" name="fPassword" />{if $forgotten_password_reset}<br/>
+		<a href="password-recover.php">{$PALANG.pUsersLogin_password_recover}</a>{/if}</td>
```

Comment:
I prefer readable templates over cleanly formatted whitespace in the HTML output, so please don't hide the `if` in the middle of two lines.